src/03-modality-agnostic-files.md

```diff
@@ -9,17 +9,17 @@ Template: `dataset_description.json` `README` `CHANGES`
-| Funding            | OPTIONAL. List of sources of funding (grant numbers)                                                                                                                                                                          |
-| ReferencesAndLinks | OPTIONAL. List of references to publication that contain information on the dataset, or links.                                                                                                                                |
-| DatasetDOI         | OPTIONAL. The Document Object Identifier of the dataset (not the corresponding paper).                                                                                                                                        |
+| Field name         | Definition                                                                                                                                                                                                                                    |
+| :----------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Name               | REQUIRED. Name of the dataset.                                                                                                                                                                                                                |
+| BIDSVersion        | REQUIRED. The version of the BIDS standard that was used.                                                                                                                                                                                     |
+| License            | RECOMMENDED. What license is this dataset distributed under? The use of license name abbreviations is suggested for specifying a license. A list of common licenses with suggested abbreviations can be found in Appendix II.                 |
+| Authors            | OPTIONAL. List of individuals who contributed to the creation/curation of the dataset.                                                                                                                                                        |
+| Acknowledgements   | OPTIONAL. Text acknowledging contributions of individuals or institutions beyond those listed in Authors or Funding.                                                                                                                          |
+| HowToAcknowledge   | OPTIONAL. A string of text containing instructions how researchers using this dataset should acknowledge the original authors. This field can also be used to define a publication that should be cited in publications that use the dataset. |
```

Comment:
@sappelhoff I believe @teonbrooks was just suggesting a minor wording change:

```suggestion
| HowToAcknowledge   | OPTIONAL. Text containing instructions how researchers using this dataset should acknowledge the original authors. This field can also be used to define a publication that should be cited in publications that use the dataset. |
```

This seems unambiguous and matches `Acknowledgements`.

Reply:
precisely. next time, i will just be more explicit, hehe. just would add the word `on` to read `Text containing instructions on how`

Reply:
Updated.

Reply:
alright, thanks for clearing that up :-)